package protoFile.afk.DianShiChengJi2802;
message DianShiChengJiRequest{
    required int32 id=1;//½ÇÉ«id
}
message DianShiChengJiResponse{
    optional string message=1;
    optional bool result=2;
}